### DISCUSSION TITLE
packwerk check --ignore-todos

### DISCUSSION THREAD

**Discussion by exterm** [Shopify/packwerk#395] [Ideas]
For editor integration through [packwerk-vscode](https://github.com/rubyatscale/packwerk-vscode), I'd love to be able to see all the violations, even the todo-listed ones. I would also like this output for downstream analysis tools so that I don't have to use [`parse_packwerk`](https://github.com/rubyatscale/parse_packwerk) which works on the todo-list files. Those files omit some information that the output of `packwerk check` includes or can include with a custom offense formatter.

If `packwerk check` had a `--ignore-todos` option, that would support my use case. Would anyone else be interested in this feature?

**Comment by zekenie** [comment]
this would be very helpful for some tooling we're working on as well

**Comment by exterm** [comment]
I'm just realizing that the naming is confusing; todos are ignored by default, so it should probably be `--no-ignore-todos` - open to suggestions

**Comment by exterm** [comment]
Found another way to get to this data - hoping to be able to get this merged: https://github.com/Shopify/packwerk/pull/397
